src/bundle.c

```diff
@@ -572,15 +573,14 @@ int remove_bundles(char **bundles)
 			  current_version,
 			  ret);
 		if (ret) {
-			fprintf(stderr, "Error: Bundle remove failed\n");
 			ret_code = ret;
 		}
 	}
 
 	if (bad > 0) {
-		fprintf(stderr, "%i bundle(s) of %i failed to be removed\n", bad, total);
+		fprintf(stderr, "%i bundle%s of %i failed to be removed\n", bad, (bad > 1 ? "s" : ""), total);
```

Comment:
Since the purpose of this PR is to update our messaging, I'll mention that I don't like the passive tense here. I'd prefer something like `failed to remove %i of %i bundle%s`

Reply:
I expected this comment to be listed first in the review, hence my explanation of my suggestions on the messaging.